validator/core/cycle.py

```diff
@@ -317,10 +327,23 @@ async def process_completed_tasks(config: Config) -> None:
             await asyncio.sleep(30)
 
 
+async def _move_back_to_looking_for_nodes(task: Task, config: Config):
+    task.status = TaskStatus.LOOKING_FOR_NODES
+    await tasks_sql.update_task(task, config.psql_db)
+
+
+async def _check_delayed_tasks(config: Config):
```

Comment:
misleading name in my opinion. Not just checking the tasks but _handle_delayed_tasks or _retry_delayed_tasks or smth?